cms/static/sass/elements/_modal-window.scss

```diff
@@ -777,4 +780,44 @@
       }
     }
   }
+
+  .edit-unit-access, .edit-staff-lock {
+    .modal-section-content {
+    @include font-size(16);
+
+    .group-select-title {
+      font-weight: font-weight(semi-bold);
+      font-size: inherit;
+      margin-bottom: ($baseline/4);
+
+      .user-partition-select {
+        font-size: inherit;
```

Comment:
We use inherit so that the children elements receive the same font size from the modal-section-content parent. Without the inherit we end up with a font size of 12 for the text inside of the modal.